src/main/java/org/beanpod/switchboard/controller/EncoderController.java

```diff
@@ -80,4 +81,15 @@ public ResponseEntity createEncoder(@RequestBody @Valid EncoderEntity encoderEnt
         }
         return ResponseEntity.ok("Encoder with serial number " + serialNumber + " Deleted");
     }
+
+    @PutMapping
+    public ResponseEntity<EncoderDTO> updateEncoder(@RequestBody EncoderDTO encoderDTO){
```

Comment:
I thought the update endpoints for encoders and decoders were part of issue #23? But these endpoints were not included in PR #89. We should look out for that in our code reviews in the future.

Reply:
the reason is written in the description of that pr

Reply:
Sorry, I missed that when i read it, thanks :)